tests/network-setups/fabric/dev/scripts/setupCCLocal.sh

```diff
@@ -19,9 +19,16 @@ cp -r ${INTEROP_CC_PATH} .
 cd fabric-interop-cc
 cp -r $PWD/../../../../../../common/protos-go contracts/interop/
 cp -r $PWD/../../../../../../core/network/fabric-interop-cc/libs/assetexchange contracts/interop/
+cp -r $PWD/../../../../../../core/network/fabric-interop-cc/libs/utils contracts/interop/
+cp -r $PWD/../../../../../../core/network/fabric-interop-cc/libs/testutils contracts/interop/
```

Comment:
Same comment as above. if it requires `go.sum` may be check-in `go.sum.local` as well? Then these test utils won't be required?